web/marketing/pages/partner-offers-modal.html

```diff
@@ -0,0 +1,61 @@
+<!-- Copyright (C) 2019 Storj Labs, Inc.
+See LICENSE for copying information. -->
+
+{{define "partnerOffersModal"}}
+<div class="modal fade p-5" id="partner-offers-modal" tabindex="-1" role="dialog" aria-labelledby="partner-offers-modal-lbl" aria-hidden="true">
+    <div class="modal-dialog modal-lg" role="document">
+        <div class="modal-content">
+            <div class="modal-header">
+                <h5 class="modal-title" id="partner-offers-modal-lbl">Create Partner Offer</h5>
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                <span aria-hidden="true">&times;</span>
+                </button>
+            </div>
+            <div class="modal-body">
+                <form>
+                    <div class="form-row">
+                        <div class="form-group col-md-4">
+                            <label for="Name">Offer Name</label>
+                            <input type="text" class="form-control" name="Name" id="Name" placeholder="May Referral" required>
+                        </div>
+                        <div class="form-group col-md-4">
+                            <label for="Description">Description</label>
+                            <input type="text" class="form-control" name="Description" id="Description" placeholder="Our test with $50 for May" required>
+                        </div>
+                        <div class="form-group col-md-4">
+                            <label for="ExpiresAt">Credit Exp Date</label>
+                            <input type="date" class="form-control" id="ExpiresAt" name="ExpiresAt" placeholder="06/01/19" required>
+                        </div>
+                    </div>
+                    <div class="form-row">
+                        <div class="form-group col-md-2">
+                            <label for="InviteeCredit">Give Credit</label>
```

Comment:
I don't think we need this column for partner offer

Reply:
Just to confirm based on our stand up today, we are keeping this and removing the AwardCredit for the partner `offerType` correct ?

Reply:
Yes